themes/vocabulary_theme/templates/page-with-toc.html

```diff
@@ -51,7 +51,7 @@ <h1>{{ this.title }}</h1>
                 </ul>
               </aside>
             {% elif this.is_child_of('/community') %}
-              <aside class="menu sidebar-menu">
+              <aside class="menu sidebar-menu" role="complementary" aria-label="Community Sidebar">
```

Comment:
`aside` already has a semantic role, unsure this is necessary. Additionally if it needs a label it should likely have a heading within it to support that.